sdk/appconfiguration/azure-appconfiguration/tests/test_azure_configuration_client.py

```diff
@@ -38,7 +41,7 @@ def __init__(self, method_name):
         if self.is_playback():
             connection_str = "Endpoint=https://fake_app_config.azconfig-test.io;Id=0-l4-s0:h5htBaY5Z1LwFz50bIQv;Secret=bgyvBgwsQIw0s8myrqJJI3nLrj81M/kzSgSuP4BBoVg="
         else:
-            connection_str = os.getenv('APP_CONFIG_CONNECTION')
+            connection_str = os.getenv('APPCONFIGURATION_CONNECTION_STRING')
```

Comment:
I guess we can remove the entire block from line 41 to line 45 now right, since we create the client inside the test